core/templates/dev/head/domain/statistics/FeedbackImprovementTaskObjectFactorySpec.ts

```diff
@@ -171,27 +173,44 @@ describe('FeedbackImprovementTaskObjectFactory', function() {
     ImprovementModalService = _ImprovementModalService_;
     ThreadDataService = _ThreadDataService_;
     FEEDBACK_IMPROVEMENT_TASK_TYPE = _FEEDBACK_IMPROVEMENT_TASK_TYPE_;
+    $httpBackend = $injector.get('$httpBackend');
   }));
 
   describe('.createNew', function() {
     it('retrieves data from passed thread', function() {
       var mockThread = {threadId: 1};
       var task = FeedbackImprovementTaskObjectFactory.createNew(mockThread);
 
+      expect(task.isObsolete()).toBe(false);
       expect(task.getDirectiveData()).toBe(mockThread);
       expect(task.getDirectiveType()).toEqual(FEEDBACK_IMPROVEMENT_TASK_TYPE);
     });
   });
 
   describe('.fetchTasks', function() {
+    afterEach(function() {
+      $httpBackend.verifyNoOutstandingExpectation();
+      $httpBackend.verifyNoOutstandingRequest();
```

Comment:
$httpbackend isn't used anywhere in the actual test right? Is this needed?

Reply:
Sorry, I forgot to remove it!